docs/src/main/sphinx/connector/ranger.rst

```diff
@@ -0,0 +1,212 @@
+=======================
+Apache Ranger connector
+=======================
+
+.. raw:: html
+
+  <img src="../_static/img/apache_ranger.png" class="connector-logo">
+
+https://ranger.apache.org/
+
+Apache Ranger is a framework to enable comprehensive data security across many platforms. Trino is one of these platforms.
+
+Requirements
+------------
+
+To connect to Apache Ranger you need:
+
+* Apache Ranger installed, up and running. Compatible with Ranger v2.1.x, 2.2.x, 2.3.x
```

Comment:
How do we verify that?

Reply:
@dprophet we should add some tests for this. @kokosing recommends something like [TestSqlStandardAccessControlChecks](https://github.com/trinodb/trino/blob/403/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestSqlStandardAccessControlChecks.java) and we need some environemnt with additional services, so we could follow [EnvMultinodeKafka](https://github.com/trinodb/trino/blob/403/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeKafka.java).